README.md

```diff
@@ -1,2 +1,70 @@
-# react-todo-list
```

Comment:
It looks like your README.md file is the same as the default React app README.md. It would be better to have more detailed information about your project inside the README.md, such as Project description, Requirements, Technologies used to build it, how to install it, etc.

Reply:
Done